extensions/vscode/webviews/homeView/src/stores/home.ts

```diff
@@ -113,15 +123,23 @@ export const useHomeStore = defineStore("home", () => {
 
   const updateParentViewSelectionState = () => {
     const hostConduit = useHostConduitService();
-    hostConduit.sendMsg({
-      kind: WebviewToHostMessageType.SAVE_SELECTION_STATE,
-      content: {
-        state: {
-          deploymentName: selectedContentRecord.value?.saveName,
-          configurationName: selectedConfiguration.value?.configurationName,
+    let state: HomeViewState = null;
+    // skip saving the selection if we have not yet
+    // selected a content record. This is probably an init
+    // sequence.
+    if (selectedContentRecord.value) {
```

Comment:
Bit confused by this change. Wouldn't we want this to update if the deployment selected was deleted for example?

Reply:
If the config selected is always part of the deployment, if that does not exist, then we wouldn't want to update the config selected (since we have no deployment record).

Reply:
Gotcha, that makes more sense to me 👍